tests/test_image/test_io.py

```diff
@@ -311,9 +357,15 @@ def test_imwrite(self):
         os.remove(out_file)
         self.assert_img_equal(img, rewrite_img)
 
-        ret = mmcv.imwrite(
```

Comment:
why the test is removed?

Reply:
Because auto_mkdir is deprecated, every file client will make directory automatically, this test is meaningless.

Reply:
got it